docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>TODO: Yes, it would help to clarify this.</p><p> </p><p>Text below : "use as a template <strong>de</strong> <span class="ql-cursor">﻿</span>model parameter" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Fixed. On the todo it seems(c.f. chat with Anselm) it's a way en enforce that you know which structure generated the data, and avoid shooting yourself in the foot. I'll add that note</p>